frontend/src/userForm.tsx

```diff
@@ -227,6 +233,12 @@ export function UserForm({id}: {id: number | 'new' | undefined}) {
                      value={user.email ?? undefined}
                      onChange={e => setUser({...user, email: e.target.value})}
             />
+            <TextField label='Användarnamn'
+                       type='username'
+                       className={classes.simpleField}
+                       value={user.username ?? undefined}
```

Comment:
Here `undefined` should be `''`. Otherwise the component can change between being controlled and uncontrolled.